certified-connectors/Azure Communication Services SMS Events/apiDefinition.swagger.json

```diff
@@ -0,0 +1,496 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ACS SMS Events",
+    "description": "Connector to receive SMS messages from the Azure Communication Services resources in your subscription.",
+    "version": "1.0",
+    "x-ms-api-annotation": {
+      "status": "Production"
```

Comment:
@lei-msft please change this to "Preview". we can move your connector to [GA ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-to-ga)when it meets our [requirements](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-to-ga#required-criteria).